src/transformers/generation/utils.py

```diff
@@ -1769,6 +1798,9 @@ def contrastive_search(
                 for more details.
             output_scores (`bool`, *optional*, defaults to `False`):
                 Whether or not to return the prediction scores. See `scores` under returned tensors for more details.
+            output_logits (`bool`, *optional*, defaults to `False`):
+                Whether or not to return the raw prediction logit scores. See `raw_logits` under returned tensors
```

Comment:
```suggestion
                Whether or not to return the raw prediction logit scores. See `logits` under returned tensors
```
`logits` is the tensor name defined in the output classes. This pattern occurs more times in this diff, make sure to correct them :)